### PR TITLE
PHOENIX-4285 Add PHERF.LOG_PER_NROWS constraint to PHERF.properties file

### PR DESCRIPTION
**Pull request by yanxinyi**

### CODE REVIEW

**Comment by yanxinyi** [comment]
Added a getProperties function in order to have functional tests.

File: phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java [hunk 1]
```diff
@@ -40,4 +44,23 @@ public void testUnknownOption() {
         exit.expectSystemExitWithStatus(1);
         Pherf.main(args);
     }
+
```
Comment (dbwong) [inline phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java]: I don't see a test for bad values of this passed in, non-integer negative etc.

Comment (yanxinyi) [reply phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java]: sad path added

File: phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java [hunk 2]
```diff
@@ -40,4 +44,48 @@ public void testUnknownOption() {
...
+    @Test
+    public void testDefaultLogPerNRowsArgument() throws Exception {
+        String[] args = {"-listFiles"};
+        Pherf pherf = new Pherf(args);
+        Properties prop = pherf.getProperties();
+        assertEquals(Long.valueOf(PherfConstants.LOG_PER_NROWS),
```
Comment (gjacoby126) [inline phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java]: Having a constant called LOG_PER_NROWS and another called LOG_PER_N_ROWS in the same class is pretty confusing, especially when one refers to a key and the other to a value. :-)

File: phoenix-pherf/src/test/java/org/apache/phoenix/pherf/ResourceTest.java [hunk 3]
```diff
@@ -51,6 +51,7 @@ public void testResourceListPropertyDirectory() throws Exception {
         PherfConstants constants = PherfConstants.create();
         assertNotNull(constants.getProperty("pherf.default.dataloader.threadpool"));
         assertNotNull(constants.getProperty("pherf.default.results.dir"));
+        assertNotNull(constants.getProperty("pherf.default.log_per_nrows"));
```
Comment (gjacoby126) [inline phoenix-pherf/src/test/java/org/apache/phoenix/pherf/ResourceTest.java]: Should use constant for the string

File: phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java (hunk 2 above)
Comment (yanxinyi) [reply phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java]: Will change `LOG_PER_N_ROWS` to `LOG_PER_NROWS_NAME`

File: phoenix-pherf/src/test/java/org/apache/phoenix/pherf/ResourceTest.java (hunk 3 above)
Comment (yanxinyi) [reply phoenix-pherf/src/test/java/org/apache/phoenix/pherf/ResourceTest.java]: used constant `PherfConstants.LOG_PER_NROWS_NAME` instead of `string`

**Comment by yanxinyi** [comment]
Thanks for the reviewing @dbwong @gjacoby126 :)

**Comment by yanxinyi** [comment]
do you have any suggestions here @gjacoby126

**Pull request merged**